src/routes/github/configuration/github-configuration-get.ts

```diff
@@ -1,7 +1,7 @@
 import { Installation } from "models/installation";
 import { Subscription } from "models/subscription";
 import { NextFunction, Request, Response } from "express";
-import { getInstallations, InstallationResults } from "../../jira/configuration/jira-configuration-get";
+import { getInstallations, InstallationResults } from "../../jira/jira-get";
```

Comment:
should be able to update this route to `"routes/jira/jira-get"`

Reply:
updated